src/web/components/SlotBodyEnd/ReaderRevenueEpic.tsx

```diff
@@ -108,25 +128,72 @@ const buildPayload = async (props: Props) => {
 			mvtId: Number(getCookie('GU_mvt_id')),
 			countryCode: props.countryCode,
 		},
-	};
+	} as Metadata; // Metadata type incorrectly does not include required hasOptedOutOfArticleCount property
```

Comment:
we should update this model - but the question is whether we want to keep maintaining automat-client or publish from dotcom-components instead...

Reply:
Thanks for looking over this! I think ultimately it makes sense for `support-dotcom-components` to be the home for the types.

If we collectively think it's worthwhile I can make an intermediate fix for `automat-client` since I have a related PR open there already.